src/components/members/detail/MemberDetail.tsx

```diff
@@ -51,7 +55,11 @@ const MemberDetail: FC<MemberDetailProps> = ({ memberId }) => {
   const router = useRouter();
   const { data: profile, isLoading, error } = useGetMemberProfileById(safeParseInt(memberId) ?? undefined);
   const { data: me } = useGetMemberOfMe();
-
+  const { data: meetingList } = useQuery({
+    queryKey: ['getMeetings'],
+    queryFn: () => getMeetings(Number(memberId) ?? undefined),
+  });
+  console.log(meetingList);
```

Comment:
```suggestion
  const { data: meetingList } = useQuery({
    queryKey: ['getMeetings', memberId], // 1. 쿼리키에 추가가 필요할 것 같아요!
    queryFn: () => getMeetings(Number(memberId) ?? undefined),
  });
```

2. memberId 도 string을 number로 캐스팅 하고(이게 타입 안맞거나 하면 NaN 되어버리거나 하는 위험이 있어서..!) 저걸 다시 어차피 getMeetings에서 string으로 바꿔서 쓰니까 애초에 getMeetings에 string으로 받아도 좋을 것 같아요!

쿼리키는 꼭 넣어주시구 이대로도 잘 동작한다면 2번은 나중에 반영해도 될 것 같아요!

Reply:
3. console.log 도 지우기

Reply:
급하게 하느라 꼼꼼하지 못했는데.. 챙겨주셔서 넘 감사합니다ㅜㅜ